<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NeutronYieldCalibration</class>
 <widget class="QWidget" name="NeutronYieldCalibration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>383</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>灵敏度标定</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <row>
      <property name="text">
       <string>#1</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>#2</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>#3</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>#4</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>#5</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>#6</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>#7</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>#8</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>#9</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>#10</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>#11</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>#12</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>#13</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>#14</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>#15</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>#16</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>#17</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>#18</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>#19</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>#20</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>#21</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>#22</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>#24</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>中子产额</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>初始活度</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QPushButton" name="pushButton_import">
       <property name="text">
        <string>导入...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_export">
       <property name="text">
        <string>导出...</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_save">
       <property name="text">
        <string>保存</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_cancel">
       <property name="text">
        <string>退出</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
